Modules/Scripted/DICOM/DICOM.py

```diff
@@ -8,6 +8,7 @@
 import slicer
 from slicer.util import settingsValue, toBool
 from slicer.ScriptedLoadableModule import *
+from slicer.translation import _
```

Comment:
```suggestion
from slicer.translation import tr as _
```

Reply:
This would be consistent with how django approach this. See https://docs.djangoproject.com/en/dev/topics/i18n/translation/

It would also allow things like 

```python
from slicer.translation import tr_noop as _
```

Reply:
Agreed!